doc/api-reference.md

```diff
@@ -204,6 +204,14 @@ Captures the portion of the input that was consumed by the node.
 
 The number of child nodes that the node has.
 
+<b><pre class="api">n.isOptional() &rarr; boolean</pre></b>
+
+`true` if the node is an iterator node and can have no or one children (? operator), otherwise `false`.
```

Comment:
I'd say "`true` if the node is an iterator node having either one children or no children"